src/parse/import.rs

```diff
@@ -7,6 +7,56 @@ use crate::{common::QuoteKind, error::SassResult, lexer::Lexer, value::Value, To
 
 use super::{Parser, Stmt};
 
+/// Searches the current directory of the file then searches in load paths directories
+/// if the import has not yet been found.
+/// [https://sass-lang.com/documentation/at-rules/import#finding-the-file](finding a file)
+/// [https://sass-lang.com/documentation/at-rules/import#load-paths](load path)
+fn find_import(file_path: &PathBuf, name: &OsStr, load_paths: &[&Path]) -> Option<PathBuf> {
+    let paths = [
+        file_path.with_file_name(name).with_extension("scss"),
+        file_path
+            .with_file_name(format!("_{}", name.to_str().unwrap()))
+            .with_extension("scss"),
+        file_path.clone(),
+        file_path.join("index.scss"),
+        file_path.join("_index.scss"),
+    ];
+
+    for name in &paths {
+        if name.is_file() {
+            return Some(name.to_path_buf());
+        }
+    }
+
+    for path in load_paths {
```

Comment:
`load_paths` could potentially be the same as `paths`, I don't think we need to search the same thing twice, or maybe we could just ignore this?

Reply:
The difference between `path` and `load_paths` is that `load_paths` is just a list of places to search for potential files that you want to import. 
We could do loop over just one slice but I decided against doing that as it would involve calculate the path combinations for each directory/file in `load path` at the start. Instead of working it out as we as we go along.

Reply:
I think the current method that lazily calculates paths makes the most sense.

Reply:
chain method should be evaluating it lazily, you can just resolve this issue, I didn't know it was different back then.